hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/metrics/StoreMetricsService.java

```diff
@@ -24,6 +24,7 @@ public interface StoreMetricsService {
     enum StoreType {
         TOPIC,
         ACCOUNT,
+        AIRDROPS,
```

Comment:
I think Michael H had a comment that we should rename this to AIRDROP for consistency

Reply:
Done 👍